README.md

```diff
@@ -75,7 +77,9 @@ This function creates a new instance of `AssetComputeClient` and calls the `.reg
     const yaml = require("js-yaml");
     const { AssetComputeClient } = require("@adobe/asset-compute-client");
 
-    const integration = yaml.safeLoad(await fs.readFile("integration.yaml", "utf-8"));
+    //If integration file is json, a private key file must also be provided
+    const integrationFilePath = "/path/to/integration/file"; // Either json or yaml format
+    const integration = AssetComputeClient.getConfiguration(integrationFilePath[, privateKeyFile]);
```

Comment:
maybe I am missing something, but where is this function inside `AssetComputeClient`?

Reply:
also after this is added to the `AssetComputeClient` class, please add a unit test for it :)

Reply:
Changed it around